doc/index.md

```diff
@@ -165,7 +165,7 @@ When listening on the network, the same API is available on a TLS socket
 Canonical RBAC.
 
 
-More details are [available here](security.md).
+More details are [available here](doc/security.md).
```

Comment:
This will break links for anyone looking at the documentation. Currently there are symlinks in place and a bug in Github's web UI where they're not followed, let's not break things further :)

Reply:
Ok, I will remove the commit.